horace_core/sqw/@DnDBase/private/sqw_eval_nopix_.m

```diff
@@ -0,0 +1,30 @@
+function obj = sqw_eval_nopix_(obj, sqwfunc, all_bins, pars)
+% SQW_EVAL_NOPIX_
+%
+% Helper function for sqw eval executed on a pixel-less object (DnD with no pixels
```

Comment:
unclear to me if dnd always has no pixels or if this is a special-case dnd with no pixels

Reply:
dnd always have not pixels. Sqw may not have pixels (questionable design solution but historical)